osgar/drivers/realsense.py

```diff
@@ -6,6 +6,7 @@
 import logging
 import threading
 import numpy as np
+import cv2
```

Comment:
It seems somehow wrong to create dependency in the `realsense` driver on `cv2` package. Have you tried mjpeg?

https://intelrealsense.github.io/librealsense/python_docs/_generated/pyrealsense2.format.html#pyrealsense2.format.mjpeg

Reply:
Unforchunetly, the mjpeg did not work and i did not find a solution. I just get following error:

$ python -m osgar.record config/test-realsense.json 
2020-06-17 12:59 root         WARNING  Environment variable OSGAR_LOGS is not set - using working directory
2020-06-17 12:59 __main__     INFO     /home/jakub/git/osgar/test-realsense-200617_105914.log
2020-06-17 12:59 osgar.drivers.realsense INFO     Found Intel RealSense D435I (S/N: 846112072840): Enabling depth stream
Traceback (most recent call last):
  File "/home/jakub/miniconda3/envs/osgar2/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/jakub/miniconda3/envs/osgar2/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/jakub/git/osgar/osgar/record.py", line 96, in <module>
    main()
  File "/home/jakub/git/osgar/osgar/record.py", line 93, in main
    record(cfg, log_prefix=prefix, duration_sec=args.duration)
  File "/home/jakub/git/osgar/osgar/record.py", line 69, in record
    with Recorder(config=config['robot'], logger=log) as recorder:
  File "/home/jakub/git/osgar/osgar/record.py", line 36, in __enter__
    module.start()
  File "/home/jakub/git/osgar/osgar/drivers/realsense.py", line 142, in start
    self.depth_pipeline.start(depth_cfg, self.depth_callback)
RuntimeError: Couldn't resolve requests